kura/org.eclipse.kura.util/src/main/java/org/eclipse/kura/util/base/ThrowableUtil.java

```diff
@@ -1,54 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2016 Eurotech and/or its affiliates and others
- *
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- * 
- *******************************************************************************/
-package org.eclipse.kura.util.base;
-
-import java.io.PrintWriter;
-import java.io.StringWriter;
-
-import org.eclipse.kura.KuraErrorCode;
-import org.eclipse.kura.KuraRuntimeException;
-
-/**
- * The Class ThrowableUtil contains all necessary static factory methods for
- * manipulating Throwable instances
- */
-public final class ThrowableUtil {
```

Comment:
Don't drop it right away. I have written an important and more useful method which I am making use of in other locations. In addition, instead of continuing to use `org.eclipse.kura.core.util.ThrowableUtil`, we can use this.